packages/vehicle-rental-overlay/package.json

```diff
@@ -18,6 +18,7 @@
     "url": "git+https://github.com/opentripplanner/otp-ui.git"
   },
   "dependencies": {
+    "@opentripplanner/base-map": "^1.1.0",
```

Comment:
I don't think this needs to be a dep since it's contained within peer deps. Perhaps removing this line and updating the version manually in the list of peer deps would be the easiest fix.

Reply:
I somehow entirely missed that peer deps were even there. Have updated and things work nicely now thank you!

Reply:
I tried this, but got serious react errors (it was loading the wrong version). When I move the dependency from peer deps to deps things work as normal... I'm not sure why